aws-greengrass-testing-features/aws-greengrass-testing-features-api/src/main/java/com/aws/greengrass/testing/features/DeploymentSteps.java

```diff
@@ -340,10 +340,23 @@ Map<String, ComponentDeploymentSpecification> parseComponentNamesAndPrepare(
             }
             overrides.component(name).ifPresent(overrideNameVersion::from);
             ComponentDeploymentSpecification.Builder builder = ComponentDeploymentSpecification.builder();
-            componentPreparation.prepare(overrideNameVersion.build()).ifPresent(nameVersion -> {
-                builder.componentVersion(nameVersion.version().value());
-            });
+
+            if (CURRENT.equals(value)) {
+                final String componentName = overrideNameVersion.build().name();
+                String componentVersion = getComponentVersionFromBackendDeployment(componentName);
+                if (componentVersion == null) {
+                    throw new IllegalStateException("Couldn't get version of component " + componentName);
+                }
+                LOGGER.info("Assume component {} has current version {}", componentName, componentVersion);
```

Comment:
debug, not info.

Reply:
Fixed in 4dcc7d1e5a10ac102d7af8bebeb81d56db672de8